src/plugins/intel_npu/src/utils/src/zero/CMakeLists.txt

```diff
@@ -2,26 +2,58 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set(TARGET_NAME openvino_npu_zero_result_parser)
+set(TARGET_NAME openvino_npu_zero_utils)
 
 file(GLOB_RECURSE SOURCES *.cpp)
+source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
 
 add_library(${TARGET_NAME} STATIC ${SOURCES})
 set_target_properties(
   ${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
 ov_add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 
-target_include_directories(
-  ${TARGET_NAME}
+add_library(openvino::npu_zero_utils ALIAS ${TARGET_NAME})
+set_target_properties(${TARGET_NAME} PROPERTIES EXPORT_NAME npu_zero_utils)
+
+target_include_directories(${TARGET_NAME}
   PUBLIC
     $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>
     $<TARGET_PROPERTY:LevelZero::NPUExt,INTERFACE_INCLUDE_DIRECTORIES>
     $<BUILD_INTERFACE:${NPU_UTILS_SOURCE_DIR}/include>)
 
+
 target_link_libraries(
   ${TARGET_NAME}
   PRIVATE openvino::runtime openvino::runtime::dev)
+
 #
 # targets install
 #
-ov_install_static_lib(${TARGET_NAME} ${NPU_INTERNAL_COMPONENT})
+ov_install_static_lib(${TARGET_NAME} ${NPU_PLUGIN_COMPONENT})
+
+ov_developer_package_export_targets(TARGET openvino::npu_zero_utils
+                                    INSTALL_INCLUDE_DIRECTORIES
+                                        $<BUILD_INTERFACE:${NPU_UTILS_SOURCE_DIR}/include>)
+
+if(TARGET ze_loader)
+    ov_developer_package_export_targets(TARGET ze_loader
+                                        INSTALL_INCLUDE_DIRECTORIES
+                                            $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>)
+
+    ov_developer_package_export_targets(TARGET utils
+                                        INSTALL_INCLUDE_DIRECTORIES
+                                            $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>)
+
+    ov_developer_package_export_targets(TARGET level-zero-ext
+                                        INSTALL_INCLUDE_DIRECTORIES
+                                            $<TARGET_PROPERTY:LevelZero::NPUExt,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
I suppose explicit `INSTALL_INCLUDE_DIRECTORIES` are not required? and includes directories for these targets are properly configured?

Reply:
done, thx